code/game/machinery/Sleeper.dm

```diff
@@ -78,6 +78,11 @@
 	max_chem = E * 20
 	min_health = -E * 25
 
+/obj/machinery/sleeper/Destroy()
+	for(var/mob/M in contents)
+		M.loc = loc

```

Comment:
`M.forceMove(get_turf(src))`

You never want to set a mob's `loc` directly, because of lighting.